examples/continuous_mastering.py

```diff
@@ -0,0 +1,38 @@
+from getpass import getpass
+
+import tamr_client as tc
+
+username = input("Tamr Username:")
+password = getpass("Tamr Password:")
+
+auth = tc.UsernamePasswordAuth(username, password)
+session = tc.session.from_auth(auth)
+
+protocol = "http"
+host = "localhost"
+port = 9100
+
+instance = tc.Instance(protocol=protocol, host=host, port=port)
+
+project = tc.project.by_name(session, instance, "MasteringTutorial")
```

Comment:
You can use `from typing import cast` to cast the project to `MasteringProject`:

`
project = cast(MasteringProject, tc.project.by_name(session, instance, "MasteringTutorial"))
`
```suggestion
project = tc.project.by_name(session, instance, "MasteringTutorial")
mp = cast(tc.MasteringProject, project)
```

but that makes the example uglier...

Reply:
@skalish are we sure that this works in practice? i.e. that `mypy` is able to infer `MasteringProject` during compile-time typechecking?

Reply:
I think I've got it now.  It makes sense to check that the project is type `MasteringProject` before passing it to the function, in which case `mypy` understands.

Reply:
Nice! forgot that `mypy` has typeguards built-in for `isinstance` checks 😄